tools/ld/eosio-ld.cpp.in

```diff
@@ -30,10 +30,13 @@ int main(int argc, const char **argv) {
 #endif
          return -1;
   } else {
-      if (!eosio::cdt::environment::exec_subprogram("wasm-ld", opts.ld_options))
+      if (!eosio::cdt::environment::exec_subprogram("wasm-ld", opts.ld_options)) {
+         std::cout << "Exit due to wasm-ld failure" << std::endl;
```

Comment:
Use ```std::cerr``` for these error messages.  I goofed on line 46, can you change that to cerr too and fix the message to not reference `eosio.wasmsdk`, it hasn't been that in 3 years.